cli/tests/integrations/test_experimental.py

```diff
@@ -301,37 +300,37 @@ def _service_start_cmd(package_name,
                        package_version=None,
                        options=None,
                        json=False):
-    return (command_base
-            + (['service', 'start'])
-            + (['--json'] if json else [])
-            + ([package_name])
-            + (['--package-version', package_version]
-               if package_version else [])
-            + (['--options', options] if options else []))
+    return (command_base +
+            (['service', 'start']) +
+            (['--json'] if json else []) +
+            ([package_name]) +
+            (['--package-version', package_version]
+             if package_version else []) +
+            (['--options', options] if options else []))
```

Comment:
Formatting change. In PEP8 multi-line expression should have each line end in a binary operation.